### PR TITLE
#28 TransactionMapper refactoring

### PR DESCRIPTION
**Pull request by kolyuchkin**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,6 +86,35 @@
...
+                        </path>
+                        <path>
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok</artifactId>
+                            <version>${lombok.version}</version>
+                        </path>
+                        <path>
+                            <groupId>org.projectlombok</groupId>
+                            <artifactId>lombok-mapstruct-binding</artifactId>
+                            <version>0.2.0</version>
```
Comment (IlyaLisov) [inline pom.xml:113]: Move this version to properties as `lombok.version` above

File: src/main/java/com/example/cqrsbankingapp/web/dto/mapper/TransactionMapper.java [hunk 2]
```diff
@@ -4,7 +4,12 @@
 import com.example.cqrsbankingapp.web.dto.TransactionDto;
 import org.mapstruct.Mapper;
 
-@Mapper(componentModel = "spring")
+import static org.mapstruct.InjectionStrategy.CONSTRUCTOR;
+import static org.mapstruct.MappingConstants.ComponentModel.SPRING;
+
+@Mapper(componentModel = SPRING,
+        uses = CardMapper.class,
+        injectionStrategy = CONSTRUCTOR)
```
Comment (IlyaLisov) [inline src/main/java/com/example/cqrsbankingapp/web/dto/mapper/TransactionMapper.java:12]: move `componentModel` to new line and closing brace to new line too
```
@Mapper(
    ...,
    ...,
    ...,
)
```

**Review by IlyaLisov — CHANGES_REQUESTED** [review]
Thanks for contributing, I will be glad if you change `componentModel="spring"` with constant in other mappers too

File: pom.xml (hunk 1 above)
Comment (kolyuchkin) [reply pom.xml]: Fixed it.

**Review by IlyaLisov — APPROVED** [review]
Looks good to merge. Thanks again!

**Comment by kolyuchkin** [comment]
image

**Pull request merged**